index.js

```diff
@@ -3,6 +3,11 @@ import postcss from 'postcss';
 import perfectionist from 'perfectionist';
 import postcssSafeParser from 'postcss-safe-parser';
 
+const SUPPORTED_SCOPES = [
```

Comment:
I actually just went ahead and looked up how to use a `Set` and I, hopefully, made the change you were looking for.